Libraries/LibGfx/Bitmap.h

```diff
@@ -160,6 +164,7 @@ class Bitmap : public RefCounted<Bitmap> {
     AlphaType m_alpha_type { AlphaType::Premultiplied };
     Core::AnonymousBuffer m_buffer;
     Function<void()> m_destruction_callback;
+    ExifOrientation m_exif_orientation;
```

Comment:
instead of making Gfx::Bitmap aware of EXIF orientation, can we construct bitmap with rotated image directly? this way, hacks like `width_considering_exif()` and `height_considering_exif()` would no longer be necessary.

Reply:
Not to my understanding, we need to retain the original bitmap because a the CSS property `image-orientation` could disable EXIF orientations entirely.

Reply:
ok, you are right, though I assume `image-orientation` property is unlikely to change frequently, so it might be reasonable to store it with orientation applied and redo decoding whenever `image-orientation` changes.